.github/workflows/ci.yml

```diff
@@ -0,0 +1,397 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
```

Comment:
```suggestion
  RUST_BACKTRACE: 1
  CARGO_INCREMENTAL: 0
```

Reply:
That is set by `.github/actions/sccache/action.yml`. I'm also not sure if it makes a meaningful difference, since we're checking out and building afresh in all the workflows.